lib/publiccloud/provider.pm

```diff
@@ -444,6 +444,16 @@ sub terraform_prepare_env {
     $self->terraform_env_prepared(1);
 }
 
+sub terraform_cmd {
+    my ($prefix, %vars) = @_;
+    my $cmd = $prefix . ' ';
+    for my $var (keys %vars) {
+        $cmd .= sprintf(q(-var '%s=%s' ), $var, $vars{$var});
+    }
```

Comment:
A one-liner suggestion without `sprintf` or `q//`:
```suggestion
    my $cmd = $prefix . join(' ', map { "-var '$_=$vars{$_ }'" } keys %vars);
```

Reply:
That's neat but personally I think using the for loop is easier to read, so I'll keep it :)